### PR TITLE
Edited developer mode documentation adding postgresql installtion and other edits

### PR DESCRIPTION
**Pull request by eomwandho**
Edited developer mode documentation adding postgresql installtion and other edits

### CODE REVIEW

File: docs/tutorials/devel/install_devmode/index.txt [hunk 1]
```diff
@@ -8,16 +8,21 @@ In order to install Geonode 2.0 in developing mode on Ubuntu 12.04 the following
 
 For Windows: (:ref:`_install_win_devmode`)
```
Comment (capooti) [inline docs/tutorials/devel/install_devmode/index.txt]: maybe Ubuntu?

File: docs/tutorials/devel/install_devmode/index.txt [hunk 2]
```diff
@@ -45,6 +50,33 @@ For Windows: (:ref:`_install_win_devmode`)
...
+
+     .. code-block:: console
+
+      $ sudo apt-get install postgresql-9.3-postgis-2.1 postgresql-9.3-postgis-scripts
+
+     Change postgres UNIX password
+
+     .. code-block:: console
+
+      $ sudo passwd -u postgres # change password expiry infromation
```
Comment (capooti) [inline docs/tutorials/devel/install_devmode/index.txt]: why do we need to set a password at all?

**Comment by capooti** [comment]
nice, please check my comments.
As you changed just one doc file, it would be useful to squash the commits as described here: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

File: docs/tutorials/devel/install_devmode/index.txt (hunk 2 above)
Comment (eomwandho) [inline docs/tutorials/devel/install_devmode/index.txt]: Postgresql doesn't ask users to set password during initial install on Ubuntu, so its necessary to create new password for postgres. Though one can also user thier default system user, but postgres will through an error, that database doesn't exist.

**Pull request merged**